nni/contrib/compression/base/compressor.py

```diff
@@ -305,7 +305,7 @@ def get_calibration_config(self) -> Dict[str, Dict[str, Dict[str, torch.Tensor |
         -------
         Dict[str, Dict[str, Dict[str, torch.Tensor | Any]]]
             The calibration config format is {module_name: {target_name: {info_key: val}}}.
-            By default, info_key contains ['scale', 'zero_point', 'quant_dtype', 'quant_scheme'] and
+            By default, info_key contains ['scale', 'zero_point', 'quant_dtype', 'quant_scheme', 'quant_bits'] and
```

Comment:
why we need quant_bits?

Reply:
it will be used in the dorefa_quantizer to compute max_output and min_output